src/java/org/apache/cassandra/cql3/QueryProcessor.java

```diff
@@ -999,6 +999,16 @@ else if (statement instanceof BatchStatement)
                 }
                 return false;
             }
+            else if (statement instanceof TransactionStatement)
```

Comment:
should batch/txn both share a interface to avoid this duplication of code?

Reply:
I think something like a `ComplexStatement` would be very nice, and I think @jacek-lewandowski has actually hinted at wanting something like it for other reasons...

Reply:
What does `Complex` mean though?  I use `Group` in this ticket as it "groups" other statements together... having a type that works with multiple statements makes sense, so wonder what other goals/reasons exist.

Reply:
testing out

```
public interface GroupCQLStatement extends CQLStatement
{
    Iterable<? extends CQLStatement> getStatements();
}
```

Reply:
If we want to really find the best possible name, it might actually be `CompositeStatement` (in the sense that batches and txns are statements made up of other elements/statements).